filters_ublock.txt

```diff
@@ -39,10 +39,9 @@ search.seznam.cz,clanky.seznam.cz##div[data-e-b-n*="advert"]
 sreality.cz##+js(nosiif, checkRods)
 super.cz##+js(ra, style, body)
 svetandroida.cz##+js(aopr, detectAdBlocker)
-titulky.com##+js(ra, style, div[id^="head"])
-titulky.com##+js(set, showBanner600, true)
-titulky.com##+js(aeld, load)
-titulky.com##+js(aopr, onload)
+titulky.com##+js(set, showBanner700, true)
+titulky.com##+js(set, showFairUser, true)
```

Comment:
Je tohle potřeba pro ten antiadb popup? Jinak bych to smazal.

Reply:
jj -> showBanner700 je potrebny na nezobrazovanie antiadb popupu. showFairUser nie je nutne potrebny - avsak bez tohto bude js konzole chyba ze showFairUser nie je definovany.

Reply:
Konkretna podmienka pre zobrazenie popupu je priamo v htmlku:
```
if (1 && !(1+document.cookie.indexOf('adbwshown=')) && (typeof showBanner700 == 'undefined' || !showBanner700)){
					 foolish_script_is_here();
```
.. kde `foolish_script_is_here` zobrazuje ten popup.

Reply:
console chybu můžeš ignorovat. Jestli není potřeba kvůli nějaké funkčnosti (řekl bych že ne) tak to jedno pravidlo smaž (showFairUser).